src/components/HomeScrollingText.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+
+export default function HomeScrollingText() {
+  useEffect(() => {
+    const element = document.querySelector('.scroll-text-animation');
```

Comment:
Pas une bonne idée la manipulation de DOM comme ça avec React
Voir les [ref](https://react.dev/reference/react/useRef#manipulating-the-dom-with-a-ref) React